src/contexts/Setup/types.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { MaybeAccount, MaybeString } from 'types';
+
+export enum SetupType {
```

Comment:
While we are here, can this be changed to a union type?

Reply:
resolved